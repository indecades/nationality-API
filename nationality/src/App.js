import './App.css';
import Nationality from './nationality';
function App() {
  return (
    <div className="App">
      <Nationality/>
    </div>
  );
}

export default App;
// notes: 
// import javascript file