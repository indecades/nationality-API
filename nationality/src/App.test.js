import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

test('renders correctly', () => { // creates snapshot test of the app component
  // asigned the var with the json format of the app component
  const tree = renderer
  .create(<App/>)
  .toJSON();
  expect(tree).toMatchSnapshot();
  });