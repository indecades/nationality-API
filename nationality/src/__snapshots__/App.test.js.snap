// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="App"
>
  <div>
    <input
      onChange={[Function]}
      value=""
    />
    <button
      onClick={[Function]}
    >
      Calculate
    </button>
    <h3>
      Nationality, 
    </h3>
    <h3>
      Probability, 
    </h3>
  </div>
</div>
`;
